packages/platform-test-suite/test/functional/platform/Identity.spec.js

```diff
@@ -237,6 +254,66 @@ describe('Platform', () => {
 
       this.timeout(850000);
 
+      it('should fail to create an identity if asset lock amount is less than minimal', async () => {
```

Comment:
Why do you think we need to test minimal amount for both asset lock proofs? Maybe we just keep only first test? My point is that tests here are very slow and we probably need to keep only high-level checks here and implement more detailed tests on lower level (integration, unit) in specific components